### PR TITLE
Locale refactoring

### PR DESCRIPTION
**Pull request by eabay**
Can you check and give some feedback?

### CODE REVIEW

File: src/js/utils/Locale.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
 // (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
-var merge = require('lodash/object/merge');
-var Cookies = require('./Cookies');
-var fallbackLocale = 'en-US';
+
+import Cookies from'./Cookies';
```
Comment (alansouzati) [inline src/js/utils/Locale.js]: I believe there is a spacing issue here

File: src/js/utils/Locale.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+
+let fallbackLocale = 'en-US';
```
Comment (alansouzati) [inline src/js/utils/Locale.js]: I believe now we could rename `fallbackLocale` to `currentLocale`

File: src/js/utils/Locale.js [hunk 3]
```diff
@@ -14,33 +15,34 @@ function normalizeLocale(locale) {
 }
 
 module.exports = {
-  getCurrentLocale: function() {
-    if (typeof module !== 'undefined' && module.exports) {
+  setFallbackLocale(locale) {
```
Comment (alansouzati) [inline src/js/utils/Locale.js]: can we rename this to `setLocale` ?

```javascript
  setLocale(locale) => currentLocale = locale
```

**Comment by alansouzati** [comment]
Nice, I like the idea of having a function to set the locale (probably in node context we will invoke that first).

I've added minor comments regarding variable and function names.

Thanks for your contribution.

**Comment by eabay** [comment]
I have made the changes regarding your comments.

**Comment by alansouzati** [comment]
Thanks for your contribution!

**Comment by eabay** [comment]
You're welcome!

**Pull request merged**